package.json

```diff
@@ -53,6 +53,7 @@
   },
   "devDependencies": {
     "@changesets/cli": "^2.17.0",
+    "@next/bundle-analyzer": "^13.2.4",
```

Comment:
We should programmatically install this in the GitHub action so it's not included in the example project.